taipy/gui/_renderers/factory.py

```diff
@@ -335,14 +335,15 @@ class _Factory:
         )
         .set_attributes(
             [
-                ("active", PropertyType.dynamic_boolean, True),
+                ("lov", PropertyType.lov),
                 ("label",),
-                ("width",),
-                ("width[mobile]",),
                 ("on_action", PropertyType.function),
+                ("selected_ids", PropertyType.single_lov),
```

Comment:
`selected_ids` should be named `selected_items` and have property type `lov_value` if it is the result of the selection of `lov`

Reply:
As we discussed this morning... that would be 'selected' for short, and yes, hold lov elements. Not ids.
The difference with 'inactive' (and ids) is that the 'inactive' is never updated from the frontend